barretenberg/cpp/src/barretenberg/stdlib/honk_recursion/verifier/verifier.test.cpp

```diff
@@ -194,15 +203,7 @@ template <typename RecursiveFlavor> class RecursiveVerifierTest : public testing
                                                                                          pairing_points[1].get_value());
         EXPECT_EQ(recursive_result, native_result);
 
```

Comment:
Manifest for native and recursive verifier are not the same anymore due to constant proof size so this check doesn't really make sense. Maybe we could subsequently find a better alternative